pkg/reconciler/reconcile.go

```diff
@@ -0,0 +1,23 @@
+package reconciler
+
+import (
+	"context"
+
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/db"
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/forman"
+
+	"go.uber.org/zap"
+)
+
+func Reconcile(logger *zap.Logger, client *db.Database, req forman.Request) forman.Result {
+	ctx := context.Background()
```

Comment:
i was thinking of keeping new context per reconcile as it will be specific to a single provider, if you think we can have only one context I can remove this one.

Reply:
No, the reasoning sounds good to me. Let's keep it with a separate context for each reconcile.